docs/plugins/keyfinder.rst

```diff
@@ -24,9 +24,11 @@ configuration file. The available options are:
   a path to the binary. If you installed the KeyFinder GUI on a Mac, for
```

Comment:
Perhaps the sentence above should read:

> The name of the program use for key analysis. You can use either 
\`KeyFinder\`_ or \`keyfinder-cli\`_.